pkg/disttask/framework/testutil/context.go

```diff
@@ -41,6 +43,10 @@ func InitTestContext(t *testing.T, nodeNum int) (context.Context, *gomock.Contro
 	defer ctrl.Finish()
 	ctx := context.Background()
 	ctx = util.WithInternalSourceType(ctx, "scheduler")
+	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/pkg/util/cpu/mockNumCpu", "return(1)"))
```

Comment:
```suggestion
	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/pkg/util/cpu/mockNumCpu", "return(8)"))
```
i remember there're tasks of currency = 4